README.md

```diff
@@ -120,6 +122,17 @@ sources:
 **Args (sorta):**
 * `meta.is_enabled` (optional): The variable(s) you would like to reference to determine if dbt should include this table in freshness tests.
 
+----
+### is_databricks_sql_warehouse ([source](macros/is_databricks_sql_warehouse.sql))
+For Databricks destinations, this macro returns `true` if the Databricks target indicates it is a SQL Warehouse. It will return return `false` if it is an All-Purpose Cluster. 
+
+***Usage:**
+```yml
+    fivetran_utils.is_databricks_sql_warehouse(target)
+```
+**Args:**
+* `target` (required): Always set this equal to "target". This will pass all the current target information to the macro.
```

Comment:
is this actually necessary to have as an argument? like if it always must be `target` it might be harmful to have this as a parameter? cc @fivetran-joemarkiewicz as the macro author

Reply:
That's a good point. Especially since the target is always accessible regardless if it is a parameter or not. We can probably remove this as a parameter.

Reply:
I tried two versions of the macro in my local Shopify to remove the argument and neither would compile with the following errors:
1. `{% macro is_databricks_sql_warehouse(target=target) %}`
    - Error: `Compilation Error` ... `'target' is undefined`
2. `{% macro is_databricks_sql_warehouse() %}`
    - Error: `Compilation Error`... `macro 'dbt_macro__is_databricks_sql_warehouse' takes not more than 0 argument(s)`

I wasn't surprised since I have had this issue in the past trying to create arg-less macros, and searching around I haven't found a solution. @fivetran-joemarkiewicz might this be why you opted in include the target object as an argument? @fivetran-jamie Do you have any ideas on how to get this to work?

Reply:
hm how did you test these? for the second one, did you remove the `target` argument from where you call the macro?

Reply:
I got it working! Updated!